pkgs/development/tools/omnisharp-roslyn/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchurl
+, mono5
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+
+  name = "omnisharp-roslyn-${version}";
+  version = "1.32.8";
+
+  outputs = [ "out" ];
+
+  src = fetchurl {
+    url = "https://github.com/OmniSharp/omnisharp-roslyn/releases/download/v${version}/omnisharp-mono.tar.gz";
+    sha256 = "0k2a4awmzb7ppll2skyzaa94n3hxqm35ffibl0sygldk3symzwgp";
+  };
+
+  buildInputs = [ makeWrapper ];
```

Comment:
Consider moving this to `nativeBuildInputs`, which is more appropriate for build-time only inputs.

Reply:
yep